doc/source/user/tutorial-svd.rst

```diff
@@ -0,0 +1,494 @@
+works, the *first index in the 3rd axis* is the red pixel data for our image. We
+can access this by using the syntax
+
+::
+
+    >>> img[:, :, 0]
+    array([[121, 138, 153, ..., 119, 131, 139],
+           [ 89, 110, 130, ..., 118, 134, 146],
+           [ 73,  94, 115, ..., 117, 133, 144],
+           ...,
+           [ 87,  94, 107, ..., 120, 119, 119],
+           [ 85,  95, 112, ..., 121, 120, 120],
+           [ 85,  97, 111, ..., 120, 119, 118]], dtype=uint8)
+
+From the output above, we can see that every value in ``img[:,:,0]`` is an
+integer value between 0 and 255, representing the level of red in each
+corresponding image pixel (keep in mind that this might be different if you
+use your own image instead of `scipy.misc.face`).
+
+As expected, this is a 768x1024 matrix::
+
+    >>> img[:, :, 0].shape
+    (768, 1024)
+
+Since we are going to perform linear algebra operations on this data, it might
+be more interesting to have real numbers between 0 and 1 in each entry of the
+matrices to represent the RGB values. We can do that by setting
+
+    >>> img_array = img / 255
+
+This operation, dividing an array by a scalar, works because of NumPy's `broadcasting`
+rules (see :ref:`array-broadcasting-in-numpy`). (Note that in real-world
+applications, it would be better to use, for example, the `img_as_float
+<https://scikit-image.org/docs/dev/api/skimage.util.html#skimage.util.img_as_float>`_
+utility function from ``scikit-image``).
+
+You can check that the above works by doing some tests; for example, inquiring about
+maximum and minimum values for this array::
+
+    >>> img_array.max(), img_array.min()
+    (1.0, 0.0)
+
+or checking the type of data in the array::
+
+    >>> img_array.dtype
+    dtype('float64')
+
+Note that we can assign each color channel to a separate matrix using the slice syntax::
+
+    >>> red_array = img_array[:, :, 0]
+    >>> green_array = img_array[:, :, 1]
+    >>> blue_array = img_array[:, :, 2]
+
+**Operations on an axis**
+
+It is possible to use methods from linear algebra to approximate an existing set
+of data. Here, we will use the `SVD (Singular Value Decomposition)
+<https://en.wikipedia.org/wiki/Singular_value_decomposition>`_ to try to rebuild an
+image that uses less singular value information than the original one, while still
+retaining some of its features.
+
+.. note::
+
+    We will use NumPy's linear algebra module, `numpy.linalg`, to
+    perform the operations in this tutorial. Most of the linear algebra
+    functions in this module can also be found in `scipy.linalg`, and
+    users are encouraged to use the `scipy` module for real-world
+    applications. However, it is currently not possible to apply linear
+    algebra operations to n-dimensional arrays using the `scipy.linalg`
+    module. For more information on this, check the
+    `scipy.linalg Reference
+    <https://docs.scipy.org/doc/scipy/reference/tutorial/linalg.html>`_.
```

Comment:
This could become
```
:doc:`tutorial/linalg`
```
although as I said elsewhere, that is a bit too generic

Reply:
If you use intersphinx format v2, I think this can be spelt ``:doc:`<scipy:tutorial/linalg>```

Reply:
that works - thanks. I must have missed the documentation for that, where did you find it? The [intersphinx documentation](https://www.sphinx-doc.org/en/master/usage/quickstart.html#intersphinx) doesn't mention that you can use that format.

Reply:
That's not the full documentation, [this is](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_mapping)

> The unique identifier can be used to prefix cross-reference targets, so that it is clear which intersphinx set the target belongs to. A link like `` :ref:`comparison manual <python:comparisons>` `` will link to the label “comparisons” in the doc set “python”, if it exists.

Reply:
Thanks!